gen/nvml/nvml.yml

```diff
@@ -82,4 +82,5 @@ TRANSLATOR:
       - {action: replace, from: "^nvmlDeviceGetGraphicsRunningProcesses$", to: "nvmlDeviceGetGraphicsRunningProcesses_v1"}
```

Comment:
What about the `_v2` and `_v3` APIs now?

Reply:
What do you mean? In this file we just make sure that the "unversioned" symbol is converted to `v1` so that we can set the function pointer correctly in `init.go`. It follows the same pattern as all the other such functions.

Reply:
Ah you're right of course. Resolving.